src/components/OrgPostCard/OrgPostCard.tsx

```diff
@@ -149,17 +134,27 @@ export default function OrgPostCard({
   ): Promise<void> => {
     const file = e.target.files?.[0];
     if (file) {
-      const base64 = await convertToBase64(file);
-      setPostFormState((prev) => ({
-        ...prev,
-        attachments: [
-          ...prev.attachments,
-          {
-            url: base64 as string,
-            mimeType: file.type,
-          },
-        ],
-      }));
+      try {
+        const { fileUrl } = await uploadFileToMinio(
```

Comment:
1. The develop-postgres branch of the API no longer uses `fileurl` therefore your mock is incorrect. Using `fileurl` in the app will cause it to malfunction.
2. You need to use `objectName` instead 
3. You also cannot use `objectName` in the preview. This is impossible. You have to use the  browser's URL object method to preview the images and the MinioUpload.ts helper function for uploading images.